data/attributes.yml

```diff
@@ -32352,6 +32352,31 @@ base_attributes:
   - broadhead_design__fixed_blade
   - broadhead_design__mechanical
   - broadhead_design__other
+- id: 4678
+  name: Bus design type
+  description: Specifies the structural and functional configuration of buses, such as double-decker, articulated, or low-floor
+  friendly_id: bus_design_type
+  handle: bus-design-type
+  values:
+  - bus_design_type__articulated
+  - bus_design_type__bi_articulated
+  - bus_design_type__double_decker
+  - bus_design_type__low_floor
+  - bus_design_type__minicoach
+  - bus_design_type__open_top
+  - bus_design_type__single_decker
+  - bus_design_type__step_entrance
+  - bus_design_type__other
```

Comment:
I would drop this attribute, as this brings multiple overlaps with the categories suggested: minicoach, double decker, articulated...